packages/components/src/components/primitives/Icon/icons/content/contentIcons.tsx

```diff
@@ -637,3 +637,22 @@ export const where_to_vote = ({ type }: Props) => {
 
   return <path d={path[type]} />;
 };
+
+export const content_copy = ({ type }: Props) => {
```

Comment:
icons are defined in alphabetical order, please move it to the right place

Reply:
Done!